app/services/error-handler.js

```diff
@@ -0,0 +1,27 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+
+export default class errorHandlerService extends Service {
+  @service('fastboot') fastboot;
+
+  handleServerSideError(error) {
+    if (!this.fastboot.isFastBoot) {return}
+
+    if (typeof FastBootRaven !== 'undefined') {
+      const req = fastboot.request;
+      const { protocol, headers } = req;
+      const host = req.hostname || req.host || headers.host || '<no host>';
+      const originalUrl = (req.originalUrl || req.url || req.path);
+      const absoluteUrl = `${protocol}://${host}${originalUrl}`;
+      // const userScope = this.userScope; // If you have an user scope you can get from service property here
+
+      FastBootRaven.withScope(function(scope) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['FastBootRaven' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5011987)